commands/databases.go

```diff
@@ -1202,6 +1215,62 @@ func buildDatabaseCreatePoolRequestFromArgs(c *CmdConfig) (*godo.DatabaseCreateP
 	return req, nil
 }
 
+// RunDatabasePoolUpdate updates a database pool.
+func RunDatabasePoolUpdate(c *CmdConfig) error {
+	if len(c.Args) < 2 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	databaseID := c.Args[0]
+	poolName := c.Args[1]
+
+	pool, err := c.Databases().GetPool(databaseID, poolName)
+	if err != nil {
+		return err
+	}
+
+	req := &godo.DatabaseUpdatePoolRequest{}
+	size, err := c.Doit.GetInt(c.NS, doctl.ArgDatabasePoolSize)
+	if err != nil {
+		return err
+	}
+	if size != 0 {
+		req.Size = size
+	} else {
+		req.Size = pool.Size
+	}
```

Comment:
This is a nice pattern for handling PUTs. We might want to look to do this in other commands where the API endpoint expects a full representation of the resource in the PUT body (i.e. doesn't act like a patch).